examples/elasticache_memcached.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: service-operator.aws/v1alpha1
+kind: ElastiCache
+metadata:
+  name: memcached13
+spec:
+  CacheSubnetGroupName: "loadtest-cluster-k8s"
+  VpcSecurityGroupIds: "sg-0581b94aa3c0db58c, sg-02b6d0034e8c2fa1b"
```

Comment:
Long term we'll have a better implementation of this using references but this works for now since you can't manage subnets with the operator.

Reply:
Sounds good